src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/SampleMain/Marketplace/MarketplaceCalls.cs

```diff
@@ -0,0 +1,134 @@
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.UnityPackage.Model;
+using Newtonsoft.Json;
+using Scripts.EVM.Remote;
+using UnityEngine;
+
+/// <summary>
+/// Marketplace sample calls for use with the api documentation.
+/// Marketplace Api: https://docs.gaming.chainsafe.io/marketplace-api/docs/marketplaceapi
+/// Token Api: https://docs.gaming.chainsafe.io/token-api/docs/tokenapi
+/// </summary>
+public class MarketplaceCalls : MonoBehaviour
+{
+    #region fields
+
+    #region Collections
+    
+    private string collectionId721 = "d588268b-8a5b-486a-8ea1-4122b510d71e";
```

Comment:
You can add `[SerializeField]` attribute to serialize these private fields

Reply:
done